doc/design/control-plane.md

```diff
@@ -334,9 +384,10 @@ The value add is not the ANA feature itself, rather what you do with it.
 
 ## NATS & Fault management
 
-We used to use NATS as a message bus within mayastor as a whole, but as since switched for gRPC for p2p communications. \
-We will continue to use NATS for async notifications. Async in the sense that we send a message, but we do NOT wait for a reply. This mechanism does not
- do any form of "consensus," retries, and the likes. Information transported over NATS will typically be error telemetry that is used to diagnose problems. No work has started yet on this subject.
+We used to use [NATS] as a message bus within mayastor as a whole, but later switched to [gRPC] for p2p communications. \
+As of today, we use [NATS] for [events](./events.md) as async notifications via the [Publish/Subscribe Model](https://en.wikipedia.org/wiki/Publish%E2%80%93subscribe_pattern). Our current mechanism does not do any form of "consensus," retries, and the likes. Information transported over NATS will typically be error telemetry that is used to diagnose problems.
+
+> NOTE: We don't have any subscribers as things stand.
```

Comment:
The cal-home stats aggregator is a subscriber IIRC.

Reply:
Indeed, done